docs/command-line-arguments.md

```diff
@@ -36,6 +36,7 @@ yarn loki test -- --port 9009
 | **`--targetFilter`**           | Regular expression for targets that should be tested.                                                                                           | _None_                                       |
 | **`--requireReference`**       | Fail stories without reference image, useful for CI.                                                                                            | _False, true for CI_                         |
 | **`--verboseRenderer`**        | Plain text renderer, useful for CI.                                                                                                             | _False, true for CI_                         |
+| **`--dockerWithSudo`**        | Run docker commands with sudo                                                                                                             | `false`                         |
```

Comment:
Seems incorrect, it's `dockerAsSudo` in the code base. I think I prefer either `dockerWithSudo` or `dockerAsSuperUser`, WDYT?